filewriter_v2/README.md

```diff
@@ -0,0 +1,387 @@
+# FileWriter V2
+
+---
+$\color{red}\text{\large{NOTE: FileWriter V2 is currently in a preview state and will change in the future.}}$
+
+Feedback is very welcome!
+
+---
+
+FileWriter writes image data and metadata to [HDF5](https://www.hdfgroup.org/solutions/hdf5/) files. Currently, there are two different formats available. V1 is a legacy format that is loosely based on [version 1.4](https://github.com/nexusformat/definitions/releases/tag/NXmx-1.4) of the _NXmx application definition_ of the [NeXus] format. V2 follows the [_NXmx application definition_](https://manual.nexusformat.org/classes/applications/NXmx.html) of NeXus Version 2022.07, with the addition of multi-channel data support[^1].
+
+| Version | Configuration | NeXus/NXmx Version | Multi-channel data support |
+|----|----|----|----|
+| V1 | SIMPLON | loosely based on [NXmx 1.4](https://github.com/nexusformat/definitions/releases/tag/NXmx-1.4) | no |
```

Comment:
I will take a closer look at version 1.4, change the wording if necessary and add the year.

Reply:
You can't tell from the source code it it was required? If that's the only thing missing (and especially if it's not clear detector/data is required) Maybe we should write "NeXus NXmx 1.4 (2016)"?

Reply:
Should we reword this to "mostly following NXmx 1.4 (2016)", "very close to NXmx 1.4 (2016)", "almost NXmx 1.4 (2016)"? XD

Reply:
I think I would avoid language like "mostly" or "very close". Wouldn't such language also apply to V2 since things like `source_name` are also technically required?

Reply:
Let's go with "NeXus NXmx 1.4 (2016)". I'll link to the (older) pdf in the legacy docs.

Reply:
I changed it in [560ea19](https://github.com/soph-dec/documentation/pull/1/commits/560ea1908a64c39f1b9d817679495d51ecd370ae).

Reply:
Thanks. But why not link to the later document [nexus-v3.2-2016-11-22-32b130a.pdf](https://github.com/nexusformat/definitions/blob/main/legacy_docs/nexus-v3.2-2016-11-22-32b130a.pdf) instead (still NXmx 1.4).

Reply:
Looking at the linked document, I just realized nxmx 1.4 was also part of NeXus v3.2. So that is another possibility (perhaps better?): "NeXus 3.2 NXmx (2016)". What do you think?

Reply:
Oh, I didn't see that. Yes, its more consistent and makes sense to write "NeXus 3.2 NXmx (2016)".
As for what document to link, there does not seem to be a difference in the NXmx definition, so I think both are fine. If you prefer the second one, I can change it. The tag "NXmx-1.4" is somewhere in between the two versions.

Reply:
Changed the link, the wording of the text above and the version name [here](https://github.com/soph-dec/documentation/pull/1/commits/05b1207e7a03334adbda2e409c0cfbdc6fee192b).